### ISSUE TITLE
Config errors are discarded

### ISSUE THREAD

**Issue by keesvanlierop** [a7ul/esbuild-node-tsc#17]
Hello there.

While implementing this tool in my project (depending currently on Node v12.x), I experienced an issue while hooking in the `esbuild-plugin-tsc` that internally uses the nullish coalescing operator (`??`), [which is only supported natively from Node >=14.0.0](https://github.com/nodejs/node/issues/33007#issuecomment-617968420).

`readUserConfig` silently ignores all errors while processing `etsc.config.js` and since the `etsc` build succeeds, it looks like everything is ok, but in fact the entire config is ignored.

I think it would be nice to either explicitly document that using the plugin is only supported from >=14.0.0., or do process the errors from reading user config. What do you think?

**Comment by a7ul** [comment]
Yes, makes sense. Can you help with a pr for this @keesvanlierop  ?
